src/components/Party/PartyModal/flows/LeaderChangeFlow.tsx

```diff
@@ -21,14 +21,16 @@ export default function LeaderChangeFlow({
 		isSuccess,
 		isError,
 	} = useChangePartyLeader({ partyId, userId });
-	const handleOnClick = () => {
-		// 리더 변경 api 호출 필요
-		changePartyLeader({ partyId, userId });
+	useEffect(() => {
 		if (isSuccess) {
 			setView('success');
 		} else if (isError) {
 			setView('failed');
 		}
+	}, [isSuccess, isError]);
+	const handleOnClick = () => {
+		// 리더 변경 api 호출 필요
```

Comment:
```suggestion
		//TODO. 리더 변경 api 호출 필요
```

TODO 주석을 쓰면 더 찾기 쉽습니다.
주석의 의미는 여러가지라서~

Reply:
넵! 주석을 구분하기 쉽도록 할 일의 경우 TODO: 주석으로 변경하겠습니다